### ISSUE TITLE
Fullscreen triggers full window on iPad

### ISSUE THREAD

**Issue by dmlap** [videojs/video.js#2350]
Tested on an iPad Mini, iOS 8.3

#### Steps to Reproduce
1. Open the sandbox and play the video
1. Tap the fullscreen button
1. Tap the native fullscreen button to exit fullscreen

Expected: the player returns to its size before fullscreen
Actual: the player fills the entire sandbox window

**Comment by gkatsev** [comment]
sounds like this is the cause of #2276

**Comment by gkatsev** [comment]
```
14:45:07 < gkatsev> I know why on ios the player is in "fullwindow" mode after exiting the fullscreen
14:45:37 < gkatsev> whether the player is fullscreen or not is set up on the player currently
14:45:49 < gkatsev> but on ios, which *still* uses the old way of doing stuff
14:45:57 < gkatsev> is inside the html5 tech
14:46:17 < gkatsev> and can't set whether the player is fullscreen yet
14:46:30 < gkatsev> so, our fullscreen handler never removes the vjs-fullscreen class
```

**Comment by gkatsev** [comment]
Weird, this didn't auto-close. Maybe because of how we merged the PR in. Anyway, fixed by #2357.
